src/Uno.UI/UI/Xaml/Media/AcrylicBrush/AcrylicBrush.skia.cs

```diff
@@ -77,15 +76,12 @@ internal override void OnPropertyChanged2(DependencyPropertyChangedEventArgs arg
 		}
 	}
 
-	// issue specific to macOSarm64 https://github.com/unoplatform/uno/issues/16853
-	static bool macOSarm64 = RuntimeInformation.IsOSPlatform(OSPlatform.OSX) && RuntimeInformation.ProcessArchitecture is Architecture.Arm64;
-
 	private void UpdateAcrylicBrush()
 	{
 		if (_isConnected)
 		{
 			// TODO: Currently we are force recreating the brush even if it exists because Composition animations aren't implemented yet
-			CreateAcrylicBrush(useCrossFadeEffect: false, forceCreateAcrylicBrush: !macOSarm64);
+			CreateAcrylicBrush(false /* useCrossFadeEffect */, true /* forceCreateAcrylicBrush */);
```

Comment:
I think named arguments are better than `/* */` comment

Reply:
you're right, I did a revert to keep git history cleaner - but I should reapply the naming.

Reply:
updated